torch/autograd/graph.py

```diff
@@ -86,7 +86,8 @@ class save_on_cpu(saved_tensors_hooks):
 
     Args:
         pin_memory (bool): If ``True`` tensors will be saved to CPU pinned memory
-                           during packing and copied to GPU asynchronously during unpacking.
+                           via asynchronous transfer to CPU during packing and
```

Comment:
nit the sentence is a bit weird with both asynchronous and asyncronously.
Maybe simply: If ``True`` tensors will use pinned memory and perform transfers between the CPU and the GPU asynchronously.